src/index.html

```diff
@@ -11,6 +11,34 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form 
+      class="search-form big-search-form"
+      method="get"
+      action="/"
+      data-qa="big"
+    >
+      <label class="search-input big-search-input">
+        <input
+          class="input big-input"
+          type="search"
+          placeholder="Try &quot;Los Angeles&quot;"
+          data-qa="keypress"
+        >
+      </label>
+    </form>
```

Comment:
I recommend to use BEM methodology for naming classes here. Apply for another form as well
```suggestion
    <form 
      class="form form--big"
      method="get"
      action="/"
      data-qa="big"
    >
      <label class="form__label form__label--big">
        <input
          class="form__input form__input--big"
          type="search"
          placeholder="Try &quot;Los Angeles&quot;"
          data-qa="keypress"
        >
      </label>
    </form>
```